CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.0] - 2022-01-27
```

Comment:
Thanks for the contribution Todd, given that we're still in < 1.0 versions, we can still make breaking changes, I suggest bumping it to 0.8.0. 

https://semver.org/#spec-item-4
<img width="736" alt="image" src="https://github.com/Gusto/ar-query-matchers/assets/653256/fef48a25-511b-43a4-96b0-7ca6509889d0">

Reply:
Thanks for the reminder about < 1.0